Tape/NAS/File-Copy Support

Jannos-443 opened the pull request:
Added:
- VeeamV10 NAS Backup Monitoring
- File Copy Monitoring
- Tape Monitoring

Jannos-443 committed:
Update PRTG-VeeamBRStats.ps1

VeeamV10 NAS Backup Support

Jannos-443 committed:
Update PRTG-VeeamBRStats.ps1

Jannos-443 committed:
Tape/NAS/File-Copy Support

Added Tape/NAS/File-Copy Backup Monitoring.

vMarkusK commented:
Wow, this looks pretty awesome. Great enhancement!

Unfortunately, I'm currently not able to verify it end-to-end. I'll try to get some folks to test it out.

vMarkusK commented:
@Jannos-443  Do you think extending the debug section is required?

Jannos-443 commented:
> @Jannos-443 Do you think extending the debug section is required?

I think the SessionObject should be enough.

Jannos-443 committed:
Changes

- BackupTransfer and BackupRead independend from Region "Backup Jobs" 
- Added "Unrecognized Tapes" to the result
- Added "Hanging Jobs" detection (Jobs running longer than 24h
- changed LimitMaxError to 0.1 to be in the green area while on zero.
- Minor Formatting changes

Jannos-443 commented:
In my envirement the Script runs up to 9 Minutes, i had to change the Timeout to 900. (15 minutes)
The Tape Job "Get-Command" is as slow as Tape ;)

vMarkusK commented:
Do you use SQL Express in your environment?

Jannos-443 commented:
SQL Standard on the Veeam Server, but maybe it's due to some larger File to Tape Jobs we are running.

vMarkusK commented:
Is there any CPU or disc contention during the report execution?

vMarkusK commented:
@Jannos-443 have you already verified your changes against v11?

I have recently created a new branch for the required changes for v11.

Jannos-443 commented:
Will check the CPU and Disk usage.

Upgrade to v11 is planned around mid march.

vMarkusK commented:
I have merged #76  into master might you please verify your additions with the new master?
The Script should be backward compatible.

Jannos-443 commented:
Sorry for the delay but our Cloud Connect Provider is upgrading on 16.4 so we have to wait for them.

adblocker-1 commented:
Count not verify in Version 12 i get no Results Backup from Tape Job
